client/src/client_sync/v23/wallet.rs

```diff
@@ -31,9 +31,11 @@ macro_rules! impl_client_v23__create_wallet {
             /// > 3. blank                   (boolean, optional, default=false) Create a blank wallet. A blank wallet has no keys or HD seed. One can be set using sethdseed.
             /// > 4. passphrase              (string, optional) Encrypt the wallet with this passphrase.
             /// > 5. avoid_reuse             (boolean, optional, default=false) Keep track of coin reuse, and treat dirty and clean coins differently with privacy considerations in mind.
-            /// > 6. descriptors             (boolean, optional, default=true) Create a native descriptor wallet. The wallet will use descriptors internally to handle address creation
+            /// > 6. descriptors             (boolean, optional, default=true) Create a native descriptor wallet. The wallet will use descriptors internally to handle address creation.
+            /// > 6. descriptors             (boolean, optional, default=true) Create a native descriptor wallet. The wallet will use descriptors internally to handle address creation.
```

Comment:
Duplicate line. But anyways lets not patch the Core docs (same comment as in another place during this review).